webpack/webpack.shared.ts

```diff
@@ -119,9 +118,9 @@ const sharedConfig: Configuration = {
       Buffer: ['buffer', 'Buffer'],
     }),
 
-    new CopyPlugin({
-      patterns: [{ from: 'node_modules/@matrix-org/olm/olm.wasm', to: '' }],
-    }),
+    // new CopyPlugin({
```

Comment:
Can safely delete this
Because of this line Webpack moved `olm.wasm` (handles matrix cryptography) near final bundle